src/Features/Admin/Categories/AddCategory/index.js

```diff
@@ -1,11 +1,23 @@
 import React, { useState } from 'react';
 
+import { useMutation } from 'react-query';
+import { useSelector } from 'react-redux';
+
 import CommonGridBasedForm from '../../../../components/CommonGridBasedForm';
 import { TEXT_FIELD } from '../../../../components/CommonGridBasedForm/FieldTypes';
+import { GetHeader } from '../../../../scripts/constants';
```

Comment:
stick to the one convention of the name either it should be camel case or Pascal case